circuits/sybil/sybilResCredentialAtomicQuerySigV2.circom

```diff
@@ -0,0 +1,12 @@
+pragma circom 2.0.0;
+
+include "sybilResCredentialAtomicQuerySigOffChain.circom";
+
+component main{public [
+                        // uniqueness claim
+                        issuerClaimNonRevState,
+
+                        // state secret claim
+                        crs,
+                        gistRoot
+]} = SybilResCredentialAtomicQuerySigOffChain(32, 32, 32);
```

Comment:
The name of this file does not look consistent with other file names

Reply:
Can you be more specific? I took the offchain circuits and just added sybilRes to them at the start. Of course, I kept the camelCase alignment.

Reply:
And if we are already talking about names, I feel that the names of the circuits aren't the best, it doesn't atomic (it is but there is no reason to specified it) or query, it's a very specific circuit.

I want to go with sybilMTP, sybilSig, wdyt?